avocado/core/result.py

```diff
@@ -234,7 +234,7 @@ def add_warn(self, state):
         """
         self.warned += 1
 
-    def add_interrupt(self, state):
+    def add_interrupt(self, state):  # Unused variable pylint: disable=W0613
```

Comment:
As mentioned earlier, you probably intended to have this squashed into the commit with similar title.